scenario/scenario_shacl.ttl

```diff
@@ -278,4 +198,78 @@ scenario:TrafficShape a sh:NodeShape ;
             sh:name "country"@en ;
             sh:order 0 ;
             sh:path scenario:country ] ;
-    sh:targetClass scenario:Traffic .
+    sh:targetClass scenario:Content .
+
+scenario:QuantityShape a sh:NodeShape ;
+    sh:closed false ;
+    sh:ignoredProperties ( rdf:type ) ;
+    sh:property [ skos:example "2" ;
+            sh:datatype xsd:unsignedInt ;
+            sh:description "Count of traffic temporary objects"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of temporaryTrafficObjects failed!"@en ;
+            sh:name "temporaryTrafficObjects"@en ;
+            sh:order 1 ;
+            sh:path scenario:temporaryTrafficObjects ],
+        [ skos:example "10" ;
+            sh:datatype xsd:unsignedInt ;
+            sh:description "Count of traffic objects"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of numberTrafficObjects failed!"@en ;
+            sh:name "numberTrafficObjects"@en ;
+            sh:order 0 ;
+            sh:path scenario:numberTrafficObjects ],
+        [ skos:example "myDriver, BestPedestrianController2000" ;
```

Comment:
macht hier ein `sh:in` Sinn?

Reply:
Ne nicht wirklich. Hatte erst gedacht ein `sh:minInclusive 0 ;` würde Sinn ergeben aber ist schon als `xsd:unsignedInt`definiert